src-data-processing/target_data_maker.R

```diff
@@ -0,0 +1,32 @@
+#' Target_data_maker
+#'
+#' A script to create data files with data available on a given date.
+
+require(tidyverse)
+require(arrow)
+# reading in the data
+file_path <- "./auxiliary-data/raw-target-data/metadata.csv"
+full_data_file <- read_csv(file_path)
+
+# reducing each clade name to there 3 character designation except recombinant
+full_data_file <- full_data_file %>%
+mutate(clade = ifelse(clade == "recombinant", "recombinant", substr(clade, 1, 3)))
+# creating a list of target dates to create versions of, starting for the first Monday in Aug 2022
+# and ending on the first Monday in Aug 2024
+target_days <- seq(from = as.Date("2022-08-01"), to = as.Date("2024-08-05"), by = 7)
+# for each date subsetting by date_submitted, grouping and creating a parquet file.
+# matching the structure of write_dataset
+file_name <- "/part-0.parquet"
```

Comment:
In this comment and the next, I suggest keeping the file name as just the file name, and using `file.path` to combine the directory with the file name.  This is a minor comment in that it doesn't impact functionality at all; it's just good practice.  One reason to use `file.path` is that it works more reliably across platforms.
```suggestion
file_name <- "part-0.parquet"
```